plans/management/commands/create_userplans.py

```diff
@@ -0,0 +1,10 @@
+from django.core.management import BaseCommand
+
+from plans.models import UserPlan
+
+
+class Command(BaseCommand):
+    help = 'Creates UserPlans for all Users'
+
+    def handle(self, *args, **options):
```

Comment:
Once you have done the as suggested in https://github.com/django-getpaid/django-plans/pull/80#issuecomment-431608635 you can exclude this from coverage via `#  pragma: no cover` comment.

```suggestion
    def handle(self, *args, **options):  # pragma: no cover
```